### PR TITLE
Remove Tracing

### PR DESCRIPTION
**Pull request by peterjan**
We never really use tracing and it's easy to temporarily re-add it to a `dev` or `debug` branch, therefor I think we should get rid of it now, right before v1.0.0.

### CODE REVIEW

**Review by ChrisSchinnerl — APPROVED** [review]

**Comment by peterjan** [comment]
since it's based on `pj/shutdown-ctx` I'll hold off on merging until that's approved

**Pull request merged**
